### PR TITLE
added good to know in `/app` based pages and layout in nested layouts

### LINKED ISSUE #51556: Docs: Mention that nested inner/nested layouts should not contain `html` or `body` tags in app router
### What is the improvement or update you wish to see?

Before starting with the app router, I went through the docs. But nowhere it was strictly mentioned that the inner layouts should not contain `html` or `body` tags.

Add a good to know section mentioning that one should not add these tags in nested layouts.

### Is there any context that might help us understand?

Because I had added `html` and `body` in my layouts, page were breaking while navigation. I only realised the error when I gave a closer look at the console. Next.js works without any compiler warnings even if `html` or `body` tags are present in the inner layouts.

To ensure that no one makes the same mistake again and that it acts as a reminder. 

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts

### PR DESCRIPTION
**Pull request by danger-ahead**
Added a good to know section mentioning that one should not add `html` or `body` tags in nested layouts under `/app` based routing.

It was not mentioned anywhere before and often one forgets this while migrating from the `/pages` based routing

Fixes #51556

### CODE REVIEW

File: docs/02-app/01-building-your-application/01-routing/02-pages-and-layouts.mdx [hunk 1]
```diff
@@ -182,6 +182,10 @@ export default function DashboardLayout({ children }) {
 }
 ```
 
+> **Good to know**:
+>
+> - Layouts defined inside a folder should not contain `html` or `body` tags.
```
Comment (leerob) [inline docs/02-app/01-building-your-application/01-routing/02-pages-and-layouts.mdx:187]:
```suggestion
> - Only the root layout can contain `html` and `body` tags.
```

Is this what you mean?

Comment (danger-ahead) [reply docs/02-app/01-building-your-application/01-routing/02-pages-and-layouts.mdx:187]: Yes.
Pushing the change.

**Review by balazsorban44 — DISMISSED** [review]
Thanks!

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 483a8d8a048be13030c59740d176c2bdf285debf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 483a8d8a048be13030c59740d176c2bdf285debf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 483a8d8a048be13030c59740d176c2bdf285debf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/01-building-your-application/01-routing/02-pages-and-layouts.mdx [hunk 2]
```diff
@@ -182,6 +182,10 @@ export default function DashboardLayout({ children }) {
 }
 ```
 
+> **Good to know**:
+>
+> - Only the root layout can contain `html` and `body` tags.
```
Comment (leerob) [inline docs/02-app/01-building-your-application/01-routing/02-pages-and-layouts.mdx:187]:
```suggestion
> - Only the root layout can contain `<html>` and `<body>` tags.
```

**Review by leerob — APPROVED** [review]

**Review by ijjk — APPROVED** [review]

**Pull request merged**